Lib/profiling/sampling/cli.py

```diff
@@ -492,6 +493,11 @@ def _add_format_options(parser, include_compression=True, include_binary=True):
         help="Output path (default: stdout for pstats, auto-generated for others). "
         "For heatmap: directory name (default: heatmap_PID)",
     )
+    output_group.add_argument(
+        "--no-browser",
```

Comment:
We probably need to make this negative for all sub profilers if someone uses `--children` so only the main one does this